### ISSUE TITLE
The ferm dependencies use the old ferm_input_list variables.

### ISSUE THREAD

**Issue by HP41** [debops/ansible-samba#9]
It uses the old vars and it's mentioned they'll be removed, it might be best to use the ferm__dependent_rules

**Comment by drybjed** [comment]
Yup, I know about the variable rename. Currently `debops.ferm` still supports old variables and it will do so for some time, until all other roles are updated. Thanks for the heads up. :-)

**Comment by HP41** [comment]
Oh nice, thanks!
